modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -98,6 +98,29 @@ a|
 The `Repeat(Trail)` operator is used to solve xref::patterns/concepts.adoc#quantified-path-patterns[quantified path patterns].
 More information can be found xref::execution-plans/operators.adoc#query-plan-repeat[here].
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+IS [NOT] :: <TYPE>
+----
+a|
+
+Added xref:syntax/expressions.adoc#type-predicate-expressions[type predicate expressions]. The available types are:
```

Comment:
two things
1. could you do the same for the constraint one below? it didn't get to be a part of the constraint PR but would be nice
2. since I got this on my PR :P
```suggestion
Added xref:syntax/expressions.adoc#type-predicate-expressions[type predicate expressions].
The available types are:
```